website/views.py

```diff
@@ -1,7 +1,59 @@
+from django.shortcuts import render, redirect
+from datetime import datetime, timedelta
+import re
+
+from django.http import JsonResponse, HttpResponse
 from django.shortcuts import render
+from django.views.decorators.csrf import csrf_exempt
+from django.contrib import messages
+
+import oscn, requests, json
+
+
+from alerts.models import Alert
 
 def index(request):
     # """View function for home page of site."""
 
     # Render the HTML template index.html with the data in the context variable
     return render(request, 'index.html')
+
+@csrf_exempt
+def form_data(request):
+    # Processes form data, requests arraignment data from api/case, and posts reminder data to api/reminder
+    # Includes option for extra phone number for additional recipient
+    case_num = request.POST['case_num']
+    year = request.POST['year']
+    county = request.POST['county']
+    phone_num = request.POST['phone_num']
+    add_num = request.POST['add_phone_num']
+    response = requests.get(
+        f"https://courtbot-python.herokuapp.com/api/case?year={year}&county={county}&case_num={case_num}"
+    )
+    resp_json = json.loads(response.content)
+
+    if resp_json.get('error', None):
+        message = resp_json['error']
+        messages.error(request, message)
+    else:
+        arraignment_datetime = resp_json.get('arraignment_datetime', None)
```

Comment:
We should check to make sure that the `arraignment_datetime` exists before making a request to setup a reminder. If the arraignment date does not exist then we should surface an error message indicating that `There was an invalid arraignment date value, please check the case for further information using steps provided at http://court.bot/#faq`